### PR TITLE
Adds two sum function

### PR DESCRIPTION
**Pull request by sophwag**
-Added two sum function
-Added two sum function tests

### CODE REVIEW

File: functions/two_sum.py [hunk 1]
```diff
@@ -1,2 +1,5 @@
 def twoSum(n, t):
-    pass
```
Comment (Jenny-PL) [inline functions/two_sum.py:2]: I love that you erased pass!

**Review by Jenny-PL — COMMENTED** [review]
nice work

File: functions/two_sum.py (hunk 1 above)
Comment (sophwag) [reply functions/two_sum.py:2]: Thanks!

File: tests/two_sum_test.py [hunk 2]
```diff
@@ -1,2 +1,37 @@
 import pytest
 from functions.two_sum import twoSum
+
+def test_returns_0_1_for_2_7_11_15_with_target_9():
+    # Arrange
+    n = [2, 7, 11, 15]
+    t = 9
```
Comment (mctagle) [inline tests/two_sum_test.py:7]: nice t

Comment (sophwag) [reply tests/two_sum_test.py:7]: thanks!

**Pull request merged**